pkg/queue/request_metric.go

```diff
@@ -123,16 +123,18 @@ func (h *requestMetricsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request
 		// If ServeHTTP panics, recover, record the failure and panic again.
 		err := recover()
 		latency := time.Since(startTime)
-		routeTag := GetRouteTagNameFromRequest(r)
+		// routeTag := GetRouteTagNameFromRequest(r)
 		if err != nil {
-			ctx := metrics.AugmentWithResponseAndRouteTag(h.statsCtx,
-				http.StatusInternalServerError, routeTag)
+			ctx := metrics.AugmentWithResponse(h.statsCtx, http.StatusInternalServerError)
+			// ctx := metrics.AugmentWithResponseAndRouteTag(h.statsCtx,
+			// http.StatusInternalServerError, routeTag)
 			pkgmetrics.RecordBatch(ctx, requestCountM.M(1),
 				responseTimeInMsecM.M(float64(latency.Milliseconds())))
 			panic(err)
 		}
-		ctx := metrics.AugmentWithResponseAndRouteTag(h.statsCtx,
-			rr.ResponseCode, routeTag)
+		ctx := metrics.AugmentWithResponse(h.statsCtx, rr.ResponseCode)
+		// ctx := metrics.AugmentWithResponseAndRouteTag(h.statsCtx,
+		// rr.ResponseCode, routeTag)
 		pkgmetrics.RecordBatch(ctx, requestCountM.M(1),
 			responseTimeInMsecM.M(float64(latency.Milliseconds())))
```

Comment:
I'm not a big fan of having code commented out like this -- can we add a comment at least indicating where there's a bunch of alternative/commented code here?

Reply:
Alternatively, being able to control this via the `tagBasedRouting` flag in `config-network` would be nice, but I realize it might be hard to reach that from `NewRequestMetricsHandler`.

Reply:
Done.